Test/cloudmake/CloudMake-ConsistentBuilds.dfy

```diff
@@ -1,4 +1,4 @@
-// RUN: %dafny /compile:0 /dprint:"%t.dprint" /autoTriggers:0 "%s" > "%t"
+// RUN: %dafny /compile:0 /dprint:"%t.dprint" /autoTriggers:0 /restartProver "%s" > "%t"
```

Comment:
Is this related to type members?

Reply:
No, only tangentially. See the commit comment for this change.